docs/master/subscriptions/client-implementations.md

```diff
@@ -184,13 +178,7 @@ const createHandler = (options) => {
         })
         .then((json) => {
           channelName =
-            !!response.extensions &&
-            !!response.extensions.lighthouse_subscriptions &&
-            !!response.extensions.lighthouse_subscriptions.channels
-              ? response.extensions.lighthouse_subscriptions.channels[
-                  operation.name
-                ]
-              : null;
+            response?.extensions?.lighthouse_subscriptions.channel ?? null;
```

Comment:
Just in case this is added and subscriptions is not (yet) enabled on the server.

```suggestion
            response?.extensions?.lighthouse_subscriptions?.channel ?? null;
```